### DISCUSSION TITLE
gruvbox-material ghostty port

### DISCUSSION THREAD

**Discussion by tacolegs2004** [ghostty-org/ghostty#2965] [Feature Requests, Ideas] [answered]
I ported [gruvbox-material](https://github.com/sainnhe/gruvbox-material) to ghostty for fun.

<img width="1512" alt="Screenshot 2024-12-14 at 2 19 01 PM" src="https://github.com/user-attachments/assets/961890ba-6be7-4d26-97f0-de8ccc57ed88" />

Here's the code:

```
palette = 0=#141617
palette = 1=#ea6962
palette = 2=#a9b665
palette = 3=#d8a657
palette = 4=#7daea3
palette = 5=#d3869b
palette = 6=#89b482
palette = 7=#ddc7a1
palette = 8=#141617
palette = 9=#ea6962
palette = 10=#a9b665
palette = 11=#d8a657
palette = 12=#7daea3
palette = 13=#d3869b
palette = 14=#89b482
palette = 15=#ddc7a1
background = 282828
foreground = d4be98
cursor-color = ffffff
selection-background = 2b2c3f
selection-foreground = 333e34
```

**Comment by jcollie** [comment, marked as answer]
Ghostty gets all of it's themes from https://github.com/mbadolato/iTerm2-Color-Schemes . We also now have a system to update weekly. So if you submit your theme to iTerm2-Color-Schemes and it gets accepted it will shortly be available in Ghostty.
